aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormField,
+  TextArea,
+  Text,
+  RadioButtonGroup,
+  FileInput,
+} from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ formProps, label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  const formats = {
+    star: { render: StarRating, border: false },
+    thumbs: {
+      render: ThumbsRating,
+      border: false,
+      label: <Text size="large">{label}</Text>,
+    },
+    textArea: { render: TextArea },
+    radioButtonGroup: { render: RadioButtonGroup },
```

Comment:
should we also add checkboxgroup?

Reply:
This wasnt in designs but I can check if we want it as an option

Reply:
Prototype. Not needed to be exhaustive at this point. Concept being proven out is the pattern of various question types. I think what we have is sufficient for proof of concept.